src/ui/zif_abapgit_gui_jumper.intf.abap

```diff
@@ -23,4 +26,12 @@ INTERFACE zif_abapgit_gui_jumper
     RAISING
       zcx_abapgit_exception.
 
+  METHODS jump_batch_input
+    IMPORTING
+      !iv_tcode      TYPE sy-tcode
+      !it_bdcdata    TYPE ty_bdcdata_tt
+      !iv_new_window TYPE abap_bool DEFAULT abap_true
```

Comment:
this variable is not used?

Reply:
it's used in `zcl_abapgit_object_dial->delete` (the only non-jump place using batch input)

Reply:
yea, its supplied, but not used in the method implementation?

Reply:
argh. yes, forgot the other call without `starting new task`...